src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -48,6 +47,9 @@ class PeerManagerActor(
     case Terminated(ref) =>
       peers -= ref.path.name
 
+    case broadcast: PeerActor.Broadcast[_] =>
+      peers.values.foreach(_.ref ! broadcast)
```

Comment:
yes, I did it this way to indicate that it is different case then getting message from other peer